M2/Macaulay2/packages/Subring/Subring.m2

```diff
@@ -0,0 +1,174 @@
+-- -*- coding: utf-8 -*-
+newPackage(
+    "Subring",
+    Version => "1.0",
+    Date => "June 6, 2023",
+    Authors => {
+	    {Name => "Francesca Gandini", Email => "fra.gandi.phd@gmail.com"},
+        {Name => "Casey Hill", Email => "casey.hill@uky.edu"},
+        {Name => "Trevor K. Karn", Email => "karnx018@umn.edu"},
+        {Name => "Miranda Moore", Email => "moor2340@umn.edu"},
+        {Name => "Christopher O'Neill", Email => "cdoneill@sdsu.edu"}},
+    Headline => "a package for subrings",
+    Keywords => {"Documentation"},
```

Comment:
You probably don't want "Documentation" as a keyword -- that's more for packages like `Macaulay2Doc` and `SimpleDoc` that actually deal with documentation.

Try picking one (or possibly more) of the mathematical keyword(s) from the headings on the [packages provided with Macaulay2](https://www.macaulay2.com/doc/Macaulay2/share/doc/Macaulay2/Macaulay2Doc/html/_packages_spprovided_spwith_sp__Macaulay2.html) documentation page.